Xwt.Gtk/Xwt.GtkBackend/PopoverBackend.cs

```diff
@@ -269,7 +269,7 @@ public void Show (Xwt.Popover.Position orientation, Xwt.Widget reference, Xwt.Re
 				popover.TransientFor.FocusInEvent += HandleParentFocusInEvent;
 			}
 
-			popover.Hidden += (o, args) => sink.OnClosed ();
+			popover.Hidden += PopoverHidden;
```

Comment:
unsuscribe on dispose

Reply:
not sure, won't popover.Destroy () raise the event and unsbscribe? @daserge could you verify?

Reply:
ahh ! yeah! sorry I thought it was the Cocoa backend 👍

Reply:
Maybe I don't understand the suggestion but this still has the issue:
![Screen Shot 2020-02-27 at 9 09 12 PM](https://user-images.githubusercontent.com/4272078/75473242-db2f2f80-59a5-11ea-9a5f-790a509b978b.png)
and if I change the sample code to `using (popover = new Popover ()) { ... popover.Show (...) }` it doesn't show up

Reply:
not `using`, the backend will be disposed when you close the window, or add another button to dispose it and see if it calls PopoverHidden there

Reply:
OK, so if I create a temporary Popover in button click handler it does call Dispose on closing.
The question is then - is the sample a legitimate usage?
It stores a variable in the field so that another Show does another subscription to Hidden.